src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```
@@ -154,6 +154,11 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <CreateDirectory Include="$(LocalInstallationOutputPath)" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <TarCommand Condition="'$(OS)' == 'Windows_NT'">C:\Windows\System32\tar</TarCommand>
```

```suggestion
    <TarCommand Condition="'$(OS)' == 'Windows_NT'">C:\Windows\System32\tar.exe</TarCommand>
```

But, I'm doubtful this is going to solve the problem. May need to use `C:\Program Files\Git\usr\bin\tar.exe` or a similar path, depending on where and how `git` is installed.